t/local/32_x509_get_cert_info.t

```diff
@@ -188,6 +188,10 @@ for my $f (keys (%$dump)) {
                   ) {
                       $ext_data =~ s{(othername:) [^, ]+}{$1<unsupported>}g;
                   }
+                  # Starting with 3.4.0 the double colon in emailAddress has been removed.
```

Comment:
It would be helpful to give here a reference to the commit of the fix in the OpenSSL repo.
(I tried a bit to look for it, without success so far)

Reply:
that's actually hilarious that a unit test change happened in the openssl repo and had this ripple effect. I wonder how many other languages had a similar problem transpire

Reply:
```suggestion
                  # Starting with 3.4.0 the double colon in emailAddress has been removed.
                  # See https://github.com/openssl/openssl/commit/de8861a7e3100
```

Reply:
@kiwiroy's suggested change is now in `32_x509_get_cert_info.t`. I added it manually.